test/index.test.js

```diff
@@ -163,61 +164,58 @@ describe('index test', () => {
             clientMock.get.yieldsAsync(null, responseMock);
             responseMock.toJSON.returns(testData);
 
-            datastore._get(testParams, (err, data) => {
-                assert.isNull(err);
+            return datastore._get(testParams).then((data) => {
                 assert.deepEqual(testData, data);
                 assert.calledWith(clientMock.get, testParams.params.id);
-                done();
             });
         });
 
-        it('gracefully understands that no one is returned when it does not exist', (done) => {
+        it('gracefully understands that no one is returned when it does not exist', () => {
             clientMock.get.yieldsAsync();
 
-            datastore._get({
+            return datastore._get({
                 table: 'pipelines',
                 params: {
                     id: 'someId'
                 }
-            }, (err, data) => {
-                assert.isNotOk(err);
-                assert.isNotOk(data);
-                done();
-            });
+            }).then((data) => assert.isNull(data));
         });
 
-        it('fails when given an unknown table name', (done) => {
+        it('fails when given an unknown table name', () =>
             datastore._get({
                 table: 'tableUnicorn',
                 params: {
                     id: 'doesNotMatter'
                 }
-            }, (err, data) => {
+            }).then(() => {
+                throw new Error('Oops');
+            }, (err) => {
```

Comment:
Why not `.catch((err) => {})` here? I just feel `throw new Error('Oops')` in the `then `is a bit confusing.